core/dom/range.js

```diff
@@ -1182,29 +1182,31 @@ CKEDITOR.dom.range = function( root ) {
 		},
 
 		/**
+		 * Change start container to its parent or new node after start container split.
+		 * If range is collapsed, perform collapse to the start and ends.
+		 *
+		 * If not collapse earlier, change end container to its parent or new node after end container split.
+		 *
+		 * Works on container if it is {@link CKEDITOR#NODE_TEXT text node}, range is collapsed or start/end is not ignored.
+		 *
```

Comment:
Cool that you've covered that. OTOH for me it's a bit too detailed and still not fully understandable without looking into code. So instead of describing each consecutive step of algorithm I'd rather try to grasp the essence of what is the result and close it in 1-2 sentences.

Reply:
It is hard to make summary of this method, but I'll try again :D